scantext/views.py

```diff
@@ -0,0 +1,395 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# http://nexb.com and https://github.com/nexB/scancode.io
+# The ScanCode.io software is licensed under the Apache License version 2.0.
+# Data generated with ScanCode.io is provided as-is without warranties.
+# ScanCode is a trademark of nexB Inc.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+# Data Generated with ScanCode.io is provided on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, either express or implied. No content created from
+# ScanCode.io should be considered or used as legal advice. Consult an Attorney
+# for any legal advice.
+#
+# ScanCode.io is a free software code scanning tool from nexB Inc. and others.
+# Visit https://github.com/nexB/scancode.io for support and download.
+
+import sys
+import tempfile
+from pprint import pprint
+
+from django.conf import settings
+from django.contrib import messages
+from django.shortcuts import render
+
+import attr
+from licensedcode import models
+from licensedcode import query
+from licensedcode.spans import Span
+from licensedcode.stopwords import STOPWORDS
+from licensedcode.tokenize import index_tokenizer
+from licensedcode.tokenize import matched_query_text_tokenizer
```

Comment:
Remove un-used imports, for example:
- import sys
- from pprint import pprint
- from django.conf import settings
- ...

You also have un-used imports within function, such as `from licensedcode.spans import Span`

Reply:
Yes, noticed them. Will remove them.